concourse/pipeline.yml

```diff
@@ -71,6 +77,10 @@ jobs:
         params:
           URL: 'https://govuk-coronavirus-business-volunteer-form-stg.cloudapps.digital/medical-equipment'
           MESSAGE: "Checks that the application deployed to staging is not serving HTTP error codes"
+      - task: run-rspec-specs
```

Comment:
Might be nice to rename this "run smoke tests" since that'll be clearer to folks in future.

Reply:
I think this would be a good idea and would keep this app in line with the vulnerable people form